code/datums/gamemode/role/role.dm

```diff
@@ -571,6 +571,12 @@
 	to_chat(antag.current, "<span class='info'><a HREF='?src=\ref[antag.current];getwiki=[wikiroute]'>(Wiki Guide)</a></span>")
 
 
+/datum/role/wizard/handle_mind_transfer(var/mob/living/new_character, var/mob/living/old_character)
+	. = ..()
+	for (var/spell/S in old_character.spell_list)
+		if (S.user_type == USER_TYPE_WIZARD)
+			new_character.add_spell(S)
```

Comment:
By looking at the implementation of `add_spell` it seems this wouldn't remove the spell from the old mind, is that the case?